applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -4,6 +4,21 @@ import { Dataview, resolveEndpoint } from '@globalfishingwatch/dataviews-client'
 import { selectWorkspace } from 'features/workspace/workspace.slice'
 import { ResourceQuery } from 'features/resources/resources.slice'
 
+// TODO should come from search or 4wings cell - not sure how to get that when set in a workspace?
+const DATASET_ID = 'carriers-tracks:v20200507'
+
+export const getUniqueDataviewId = (dataview: Dataview) => {
```

Comment:
I suggest also adding the generator type to the uid, which could help in debugging situations.